build-system/babel-plugins/babel-plugin-transform-jss/index.js

```diff
@@ -44,9 +45,28 @@ module.exports = function ({types: t, template}) {
     return filename.endsWith('.jss.js');
   }
 
-  function compileJss(JSS) {
-    const jss = create();
-    jss.setup(preset());
+  const seen = new Set();
+  function compileJss(JSS, filename) {
+    const filehash = crypto
+      .createHash('sha256')
+      .update(filename)
+      .digest('base64')
```

Comment:
This is important: base64 has both upper and lowercase chars, but CSS names are case-insensitive. Two alternatives:
 1. Call `toLowerCase()` on the result. We'd lose ~40% of entropy and make collisions somewhat more likely, but probably still ok with our population.
 2. Use a non-case-sensitive digest, such as HEX. That's what git does.

Reply:
I didn't know css is case-insensitive. Switching to hex now!